changelog/139.breaking.rst

```diff
@@ -0,0 +1,13 @@
+A series of functions were added from `sunkit_image.asda` by declassing the code.
+Previously, if one has to make objects of class ``Asda`` and ``Lamb_Oseen`` to implement asda.
+Now, all the functions are now declassed, now no object creation is required
+to implement any functionality of asda.
+Following functions were declassed:
+    1. ``gen_vel``
```

Comment:
These function names make little sense now when outside of the class so they will need to be removed.

Reply:
Yeah, will add that in TODO 👍 ... I was thinking to give better names like `generate_velocity` rather than `gen_vel`.